features/step_definitions/create_assignment_steps.rb

```diff
@@ -16,12 +16,11 @@
 After do
   RSpec::Mocks.teardown
   WebMock.disable!
-  FileUtils.rm_rf('assignment-repos')
+  # FileUtils.rm_rf('assignment-repos')
```

Comment:
Would you be able to explain the reasoning behind commenting this line?

Reply:
Fixed